collectors/bzimport/convertors.py

```diff
@@ -91,6 +94,9 @@ def groups_write(self):
         """
         appropriate write LDAP groups
         """
+        if self.bug.get("groups", []) == ["redhat"]:
+            return [settings.INTERNAL_WRITE_GROUP]
```

Comment:
Same here. As the mapping from Bugzilla to `["public", "internal", "embargoed"]` is used on two places, I would place it into a separate method and just call it from both `groups_read` and `groups_write` with an appropriate substitution.